### PR TITLE
PPF-435 Add scope to client

### PR DESCRIPTION
**Pull request by grubolsch**
### Added
Add scope (widgets / sapi3 / entry api) to Keycloak client

---
Ticket: https://jira.uitdatabank.be/browse/PPF-435

### CODE REVIEW

File: app/Keycloak/Service/ApiClient.php [hunk 1]
```diff
@@ -48,4 +49,22 @@ public function createClient(Realm $realm, Integration $integration): UuidInterf
 
         return $id;
     }
+
+    public function addScopeToClient(Realm $realm, UuidInterface $clientId, UuidInterface $scopeId): void
```
Comment (LucWollants) [inline app/Keycloak/Service/ApiClient.php:53]: When creating the client in Keycloack it is not possible to set the correct scope?

Comment (grubolsch) [reply app/Keycloak/Service/ApiClient.php:53]: I am afraid not, that is also not possible in the UI of keycloak, you must first create a client to add scopes.
You can set default scopes in the general config, but given that it varies which role we want, we cannot use this.

Here you can see the fields we can sent: https://www.keycloak.org/docs-api/latest/rest-api/index.html#ClientRepresentation

Comment (LucWollants) [reply app/Keycloak/Service/ApiClient.php:53]: It's not that the fields `defaultClientScopes` or `optionalClientScopes` can hold the scopes?

Comment (grubolsch) [reply app/Keycloak/Service/ApiClient.php:53]: I just tried and this does not work.
I also did some additional research and all google post say the same, you have to call the extra api endpoint after creating the client. This is also how it works in the GUI of keycloak.

**Review by LucWollants — APPROVED** [review]

**Pull request merged**